user-registration-app/src/app/views/live-view/live-view.component.html

```diff
@@ -148,7 +148,7 @@
         <div>
           <p class="question"><i class="fas fa-plus icon-rotate"></i>Where did the old website go?
           </p>
-          <p class="FAQanswer">It's right <a href="https://hackpsu18.firebaseapp.com">here!</a></p>
+          <p class="FAQanswer">It's right <a href="https://app.hackpsu.org">here!</a></p>
```

Comment:
I assume by old website here, we mean the current index.html rather than last year's hackathon page, right?

Reply:
Correct, is it not sending to the current index.html?

Reply:
Nope, it's good. I was just unsure of the wording at first.